azure_functions_worker/loader.py

```diff
@@ -239,3 +242,18 @@ def index_function_app(function_path: str):
                          f"{script_file_name}.")
 
     return app.get_functions()
+
+
+def get_fx_raw_bindings(indexed_function, function_info):
+    # If the flag is True, we know that:
+    # 1. Library is imported
+    # 2. At least one binding is a defined deferred binding type
+    # 3. DEFERRED_BINDINGS_REGISTRY is not None
+    if bindings.meta.DEFERRED_BINDINGS_ENABLED:
```

Comment:
This should be an explicit method call - `is_deferred_binding_enabled` - also - this isn't a constant - this method captures it well.